doc/sphinx/language/core/inductive.rst

```diff
@@ -1823,10 +1823,10 @@ The reduction for fixpoints is:
 .. math::
    (\Fix~f_i \{F\}~a_1 …a_{k_i}) ~\triangleright_ι~ \subst{t_i}{f_k}{\Fix~f_k \{F\}}_{k=1… n} ~a_1 … a_{k_i}
 
-when :math:`a_{k_i}` starts with a constructor. This last restriction is needed
-in order to keep strong normalization and corresponds to the reduction
-for primitive recursive operators. The following reductions are now
-possible:
+when the structural argument :math:`a_{i}` starts with a constructor. 
```

Comment:
was `k_i` wrong?

Reply:
In l1824 just above $k_i$ is the last argument, but the structurally smaller argument is not always the last ?

Reply:
above:
~~~
For doing this, the syntax of fixpoints is extended and becomes
[...]
where are positive integers. Each represents the index of parameter of , on which is decreasing.
~~~
> the structurally smaller argument is not always the last ?

it's the last one that matters for the reduction rule, extra arguments are just the context of the reduction
eg `plus 0 m` reduces because the head `plus 0` reduces